packages/create-sitecore-jss/src/templates/react/package.json

```
@@ -0,0 +1,122 @@
+{
+  "name": "jss-sample-react",
+  "description": "Application utilizing Sitecore JavaScript Services and React (create-react-app).",
+  "version": "20.0.0-canary.77",
```

```suggestion
  "version": "20.0.0-canary",
```
can also remove specific canary version here and below